.circleci/config.yml

```diff
@@ -0,0 +1,85 @@
+version: 2
+jobs:
+
+  build_demo_0:
+
+    docker:
+      - image: circleci/python:3.7.0
+
+    steps:
+
+      - checkout
+      - setup_remote_docker
+
+
+      - run:
+          name: compose
+          command: |
+            cd stand
+            docker-compose up
+          background: true
+
+      - run:
+          name: generator
+          command:  |
+            docker build -t generator ./Generator
+            docker run --network container:stand_clickhouse_1 generator
+
+          background: true
+
+      - run:
+          name: image
+          command:  |
+            docker build -t image ./Image
+            docker run --network container:stand_grafana_1: image
+
+      - run:
+          name: copy image
+          command:  |
+            docker cp image:/snapshots/output.png stand/snapshots/output.png
+
+
+      - store_artifacts:
+          path: stand/snapshots
+
+
+
+
+  build_demo_2:
+
+        docker:
+           - image: circleci/python:3.7.0
+
+        steps:
+
+          - checkout
+          - setup_remote_docker
+
+          - run:
+              name: compose
+              command: |
+                cd demo2
+                ls
+                docker-compose up
+              background: true
+
+
```

Comment:
please remove unnecessary blank lines. 1 `\n` is fine to highlight logic, 2 is ok to separate blocks, but 3 and more are no-no.